DigitalLearningSolutions.Data/Models/Courses/CourseStatistics.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace DigitalLearningSolutions.Data.Models.Courses
+{
+    using System;
+
+    public class CourseStatistics
+    {
+        public int CustomisationId { get; set; }
+        public int CentreId { get; set; }
+        public bool Active { get; set; }
+        public bool AllCentres { get; set; }
+        public bool AspMenu { get; set; }
+        public DateTime? ArchivedDate { get; set; }
+        public string ApplicationName { get; set; }
+        public string? CustomisationName { get; set; }
+        public string CourseName => string.IsNullOrWhiteSpace(CustomisationName)

```

Comment:
Could we group the derived properties together to find these easier when scanning over the model (and separate these from normal fields)?